### ISSUE TITLE
Getting blank page when I select certain candidates within Recruitment tab

### ISSUE THREAD

**Issue by albertrajan** [orangehrm/orangehrm#551]
Hi, I'm getting a blank page when I click on certain candidates within the Recruitment tab. I'm on version 4.3.2. Please can you assist?

This is the error seen:
[13-Aug-2019 07:52:59 America/Los_Angeles] PHP Fatal error:  Uncaught Error: Call to a member function getFullName() on boolean in /opt/bitnami/apps/orangehrm/htdocs/symfony/plugins/orangehrmRecruitmentPlugin/lib/service/CandidateHistoryService.php:140
Stack trace:
#0 /opt/bitnami/apps/orangehrm/htdocs/symfony/plugins/orangehrmRecruitmentPlugin/lib/service/CandidateHistoryService.php(77): CandidateHistoryService->getDescriptionForScheduleInterview(Object(CandidateHistory))
#1 /opt/bitnami/apps/orangehrm/htdocs/symfony/plugins/orangehrmRecruitmentPlugin/lib/service/CandidateHistoryService.php(44): CandidateHistoryService->getCandidateHistoryDescription(Object(CandidateHistory))
#2 /opt/bitnami/apps/orangehrm/htdocs/symfony/plugins/orangehrmRecruitmentPlugin/lib/service/CandidateHistoryService.php(31): CandidateHistoryService->getCandidateHistoryRecord(Object(CandidateHistory))
#3 /opt/bitnami/apps/orangehrm/htdocs/symfony/plugins/orangehrmRecruitmentPlugin/modules/recruitment/actions/addCandidateAction.class.php(113): CandidateHistoryService->getCandidateHistoryList(Obj in /opt/bitnami/apps/orangehrm/htdocs/symfony/plugins/orangehrmRecruitmentPlugin/lib/service/CandidateHistoryService.php on line 140

**Comment by albertrajan** [comment]
I think the issue started when I deleted the Hiring Manager. I have fixed it as below for those candidates where I'm getting blank pages:

mysql> select candidate_id, interview_id, interviewers from ohrm_job_candidate_history where candidate_id=10;
+--------------+--------------+--------------+
| candidate_id | interview_id | interviewers |
+--------------+--------------+--------------+
|           10 |         NULL | NULL         |
|           10 |         NULL | NULL         |
|           10 |         NULL | NULL         |
|           10 |            6 | 3_           |
|           10 |            6 | NULL         |
+--------------+--------------+--------------+
5 rows in set (0.00 sec)

mysql> delete from ohrm_job_candidate_history where candidate_id=10 and interview_id=6;
Query OK, 2 rows affected (0.00 sec)

mysql> select candidate_id, interview_id, interviewers from ohrm_job_candidate_history where candidate_id=10;
+--------------+--------------+--------------+
| candidate_id | interview_id | interviewers |
+--------------+--------------+--------------+
|           10 |         NULL | NULL         |
|           10 |         NULL | NULL         |
|           10 |         NULL | NULL         |
+--------------+--------------+--------------+
3 rows in set (0.00 sec)

So basically, deleted any row where interview_id was not NULL.
